src/shared/components/alterationsTableUtils.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+    ExtendedMutationTableColumnType,
+    MutationTableColumn,
+} from 'shared/components/mutationTable/MutationTable';
+import { getServerConfig } from 'config/config';
+import { Column } from 'shared/components/lazyMobXTable/LazyMobXTable';
+export function adjustVisibility(
+    tableColumns: Record<string, Column<any>>,
+    namespaceColumns: string[],
+    visibleColumnsProperty: string,
+    showNamespaceColumns: boolean
+) {
+    if (!visibleColumnsProperty) {
+        return;
+    }
+    const visibleColumns = visibleColumnsProperty.split(',');
+    Object.keys(tableColumns).forEach(column => {
```

Comment:
you don't have to update this but _.forIn(...) will allow you to iterate over members of object and give you both key and value.